Cargo.toml

```diff
@@ -31,7 +31,7 @@ panic = ["alloc"]
 panic-handler = ["print", "panic"]
 panic-handler-dummy = []
 system-alloc = []
-default = ["unwinder", "dwarf-expr", "hide-trace", "fde-phdr", "fde-registry"]
+default = ["unwinder", "dwarf-expr", "hide-trace", "fde-phdr", "fde-registry", "spin"]
```

Comment:
spin could be removed since fde-phdr will enable libc

Reply:
Done.